### ISSUE TITLE
Twilio

### ISSUE THREAD

**Issue by RealChrisSean** [airbytehq/airbyte#28865]

**Comment by aazam-gh** [comment]
I'd like to work on this for the Hackathon :D

**Comment by RealChrisSean** [comment]
This has been assigned to you :)

**Comment by aazam-gh** [comment]
@RealChrisSean I'm gonna need the sandbox for this connector to pass some of the tests :)

**Comment by RealChrisSean** [comment]
Sure thing. It may take a few days if not until next week. Please feel free to work on another issue while we wait on this. :D

**Comment by RealChrisSean** [comment]
We should get this for you by mid this week at the latest :D

**Comment by aazam-gh** [comment]
Sweet! I can get it merged quick then

**Comment by RealChrisSean** [comment]
Awesome!!

**Comment by nishp77** [comment]
Can I work on this? Though would require some help on python CDK currently reading out on how to create Low-code CDK via blogs and docs.

**Comment by justinbchau** [comment]
You got it! @nishp77 

You can refer to [our docs](https://docs.airbyte.com/) for information on the low-code CDK but please ask here or in Slack if you have any further questions!

**Comment by Katyayani2610** [comment]
@justinbchau will you please assign me this issue

**Comment by natikgadzhi** [comment]
We'll use this to track conversion, /cc @katmarkham and @lazebnyi

**Comment by octavia-squidington-iii** [comment]
Zendesk ticket #[5815](https://airbyte1416.zendesk.com/agent/tickets/5815) has been linked to this issue.
